rust/helpers.c

```diff
@@ -615,6 +616,18 @@ unsigned int rust_helper_NF_QUEUE_NR(unsigned int n)
 }
 EXPORT_SYMBOL_GPL(rust_helper_NF_QUEUE_NR);
 
+void rust_helper_mmget(struct mm_struct *mm)
+{
+        mmget(mm);
+}
+EXPORT_SYMBOL_GPL(rust_helper_mmget);
+
+void rust_helper_mmput(struct mm_struct *mm)
```

Comment:
`mmput` is already exported by the C side, there is no need to reexport it here.

Reply:
yes i will delete this part.